packages/bootstrap-4/test/__snapshots__/dateTimeWidget.test.tsx.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DateTimeWidget simple 1`] = `
+<div
+  className="mb-0 form-group"
+>
+  <label
+    className="form-label"
+    htmlFor="_id"
+  >
+    Some simple label
+    *
+  </label>
+  <input
+    autoFocus={true}
+    className="form-control"
+    disabled={true}
+    id="_id"
+    multiple={false}
+    name="nodejs"
+    onBlur={[Function]}
+    onChange={[Function]}
+    onFocus={[Function]}
+    required={true}
+    type="color"
```

Comment:
`type=color` for date time widget doesn't seem right. I know this comes from the default test data, but we should override these on a case by case basis to ensure that what we're testing is as close to real life scenario as possible

Reply:
umm, yeah

Should I try overriding with `null` or `undefined`?
cause in the `DateTimeWidget` component, there is already a `type=` prop hard coded

Reply:
Ah, I get it now. Yeah, this is silly, we shouldn't have to be passing in the type prop. What is the point of having a custom component if it can't even decide for itself what type HTML element it should be.

It think we should do this:
https://github.com/MLH-Fellowship/react-jsonschema-form/pull/37#pullrequestreview-436793744

Reply:
yea cool 👍

Reply:
Looks like Ashwin is already on it: https://github.com/rjsf-team/react-jsonschema-form/pull/1826